libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/msd/MsdStoreConnection.java

```diff
@@ -16,48 +16,52 @@
 
 package com.yahoo.athenz.common.server.msd;
 
-import com.yahoo.athenz.msd.Workload;
+import com.yahoo.athenz.msd.DynamicWorkload;
 import com.yahoo.athenz.msd.WorkloadOptions;
+import com.yahoo.athenz.msd.Workloads;
 
 import java.io.Closeable;
 import java.util.ArrayList;
-import java.util.List;
 
 /**
  * Storage interface for storing MicroSegmentation Daemon data
  */
 public interface MsdStoreConnection extends Closeable {
 
     /**
-     * putWorkload stores the Wordload value into the underlying storage that can be an RDMS or NoSQL
+     * putDynamicWorkLoad stores the Workload value into the underlying storage that can be an RDMS or NoSQL
      * @param workload workload to be stored in the underlying storage
      * @param options workload options to be passed to the api
      */
-    default public void putWorkLoad(Workload workload, WorkloadOptions options) {
+    default void putDynamicWorkLoad(DynamicWorkload workload, WorkloadOptions options) {
     };
 
     /**
      * getWorkloadsBySvc looks up all workloads for the requested service in the MSD storage
      * @param domain of the service to look up
      * @param service name of the service to look up
-     * @return a List of Workload
+     * @return Workloads object containing static and dynamic workloads
      */
-    default public List<Workload> getWorkloadsBySvc(String domain, String service) {
-        return new ArrayList<>();
+    default Workloads getWorkloadsBySvc(String domain, String service) {
+        Workloads workloads = new Workloads();
+        workloads.setDynamicWorkloadList(new ArrayList<>());
+        return workloads;
     }
 
     /**
      * getWorkloadsByIp looks up all workloads for the requested ip in the MSD Storage
      * @param ip to lookup
      * @return a List of Workload
      */
-    default public List<Workload> getWorkloadsByIp(String ip) {
-        return new ArrayList<>();
+    default Workloads getWorkloadsByIp(String ip) {
+        Workloads workloads = new Workloads();
+        workloads.setDynamicWorkloadList(new ArrayList<>());
```

Comment:
shouldn't we set the static list also to empty list since the rdl has it as mandatory?